### PR TITLE
Tim single reminder

### PR DESCRIPTION
**Pull request by tbrandle**
## Purpose

to only show titles, unless a title is clicked.  On click, show reminder body, title, and date

## Approach

Ember docs, Taylor, Charlie, Brenna

### Learning

(ember docs)[https://guides.emberjs.com/v2.7.0/tutorial/subroutes/]

### Follow-up tasks

closes issue #3 
start Issue #4  

### Screenshots
[link](http://recordit.co/JE3ovdwtZu)

### CODE REVIEW

**Comment by tbrandle** [comment]
@martensonbj @Tman22

File: tests/unit/routes/reminders/reminder-test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:reminders/reminder', 'Unit | Route | reminders/reminder', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
```
Comment (Tman22) [inline tests/unit/routes/reminders/reminder-test.js]: Delete this file.

File: app/templates/reminders.hbs [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <h1>remEMBER: SPLINTER</h1>
 
 {{#each model as |reminder|}}
+{{#link-to 'reminders.reminder' reminder}}
```
Comment (Tman22) [inline app/templates/reminders.hbs]: Fix spacing plz

**Comment by tbrandle** [comment]
@martensonbj @Tman22

**Comment by martensonbj** [comment]
Niiiice

**Pull request merged**